_posts/2022-08-14-data-pipelines-production-ready-great-expectations.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Make Your Trino Data Pipelines Production Ready Using Great Expectations"
+author: "Brian Olsen, Brian Zhan"
+excerpt_separator: <!--more-->
+---
+
+Great Expectations is an open source Python library that allows you to write
```

Comment:
I feel like you need some kind of hook to start things off. Ask a question, make a value statement, that sort of thing. The intro sentence being a definition is a little dry and doesn't motivate readers to keep going.